src/pages/login/Login.tsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect } from "react";
+import { auth, signInWithGoogle } from "../../firebase/auth/auth";
+import { useAuthState } from "react-firebase-hooks/auth";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faGoogle } from "@fortawesome/free-brands-svg-icons";
+import { faChevronRight } from "@fortawesome/free-solid-svg-icons";
+import {
+    StyledLoginContainer,
+    StyledLoginBox,
+    StyledTextContainer,
+    StyledTitleRowContainer,
+    StyledGoogleLoginButton,
+    StyledParagraph,
+} from "../../styles/style-components/Login.styles";
+
+function Login() {
+    const [user, loading, error] = useAuthState(auth);
+
+    useEffect(() => {
+        if (loading) {
+            return;
+        }
+        if (user) {
+            console.log("user active");
+        }
+    }, [user, loading]);
+
+    return (
+        <>
+            <StyledLoginContainer>
+                <StyledLoginBox>
+                    <StyledTextContainer>
+                        <StyledTitleRowContainer>
+                            <h1>Sign in</h1>
+                            <a href="#">I don&apos;t have an account</a>
```

Comment:
Why are there Predefined Character Entities?

Reply:
<img width="767" alt="Screenshot 2023-04-27 at 17 51 32" src="https://user-images.githubusercontent.com/95958816/234934065-1ca9629f-5ed9-4f85-a81a-99ec50b51cbf.png">


To fix this issue